dracut/30ignition/99-tpm-mode.rules

```diff
@@ -0,0 +1,2 @@
+KERNEL=="tpm[0-9]*", MODE="0660"
```

Comment:
Hmm, why do we need this? The default mode is 0600 but we run as root anyway, right?

Reply:
I can drop this but I ran into errors where the `swtpm` devices from `qemuexec` didn't have the right permissions and would die without this rule.